### PR TITLE
Remove sendifunverified logic

### PR DESCRIPTION
**Pull request by vbudhram**
The auth-server now defaults to sending emails `sendEmailIfUnverified=true`, content server does not need to send over this.

### CODE REVIEW

**Comment by vladikoff** [comment]
@vbudhram what other changes do we need for this?

**Comment by vbudhram** [comment]
@vladikoff Nope, just wanted to wait for https://github.com/mozilla/fxa-js-client/pull/219. This is ready now, r?

**Comment by vladikoff** [comment]
@vbudhram why was this waiting for fxa-js-client? do we need to include a new version of the js client in the PR?

File: app/scripts/models/user.js [hunk 1]
```diff
@@ -344,19 +343,6 @@ define(function (require, exports, module) {
     signInAccount (account, password, relier, options) {
       return account.signIn(password, relier, options)
         .then(() => {
-          const isSignUp =
```
Comment (vladikoff) [inline app/scripts/models/user.js]: 👍

**Comment by vbudhram** [comment]
@vladikoff This change is backwards compatible with current js client, but we should still update content-server once we tag next version of client.

**Comment by vladikoff** [comment]
👍

**Pull request merged**